app/controllers/admin/project_logs_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::ProjectLogsController < Admin::AdminController
+  def destroy
+    @log = ProjectLog.find(params[:id])
+    @step = ProjectStep.find(@log.project_step_id)
+    @loan = Loan.find(@step.project_id)
+
+    if @log.destroy
+      redirect_to admin_loan_path(@loan), notice: I18n.t(:notice_deleted)
```

Comment:
Yeah, that behavior might actually be browser-dependent. At least for me, on chrome beta channel for mac, it redirects me back to however far down I had been scrolled when I hit delete, but the project logs are hidden again instead of open. In any case, making a separate issue as a tweak is probably a good plan, finding a solution that's compatible with the tabs might be tricky.